extendedcopy.go

```diff
@@ -176,11 +177,9 @@ func findRoots(ctx context.Context, storage content.GraphStorage, node ocispec.D
 }
 
 // FilterAnnotation will configure opts.FindPredecessors to filter the predecessors
-// whose annotation matches a given regex pattern. For performance consideration,
-// when using both FilterArtifactType and FilterAnnotation, it's recommended to call
```

Comment:
We still need these comments as per the offline discussion.

Reply:
Added them back :p